phpunit.xml.dist

```diff
@@ -1,20 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?>
-
-<phpunit bootstrap="./tests/bootstrap.php" colors="true">
-    <testsuites>
-        <testsuite name="GeoIP2 Test Suite">
-            <directory suffix="Test.php">./tests/GeoIp2/Test/</directory>
-        </testsuite>
-    </testsuites>
-
-    <filter>
-        <whitelist>
-            <directory suffix=".php">./src/GeoIp2/</directory>
-        </whitelist>
-    </filter>
-
-    <logging>
-        <log type="coverage-clover" target="build/logs/clover.xml"/>
-    </logging>
-
+<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" bootstrap="./vendor/autoload.php" colors="true" xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/10.4/phpunit.xsd" cacheDirectory=".phpunit.cache">
+  <testsuites>
+    <testsuite name="GeoIP2 Test Suite">
+      <directory suffix="Test.php">./tests/GeoIp2/Test/</directory>
+    </testsuite>
+  </testsuites>
+  <logging/>
```

Comment:
Do we need this tag if it doesn't configure anything ?

Reply:
These changes are the result of `phpunit --migrate-configuration`. I didn't modify the file by hand.